src/utils/env-utils.ts

```diff
@@ -7,6 +7,8 @@ declare global {
     }
 }
 
+const ENABLED = 'true';
```

Comment:
Thought about making an `enum` but since this isn't _really_ going to check for multiple values and only cares if it is `true` decided just a plain old const was good.